assembly/assembly-main/src/assembly/bin/che.sh

```diff
@@ -493,9 +493,12 @@ call_catalina () {
     fi
   fi
 
+  ### Initialize default JVM arguments to run che
+  [ -z "${JAVA_OPTS}" ]  && JAVA_OPTS="-Xms256m -Xmx2048m  -Djava.security.egd=file:/dev/./urandom"
```

Comment:
previous settings were 
 -Xms256m -Xmx1024m -server

with your change it increases to 2048m